### PR TITLE
doc: fast_pair: Allow hiding UI indication

### PR DESCRIPTION
**Pull request by alstrzebonski**
Updates release notes and Fast Pair documentation.

Jira: NCSDK-14839

### CODE REVIEW

**Comment by alstrzebonski** [comment]
This PR is a documentation update. It is meant to be merged after this PR: https://github.com/nrfconnect/sdk-nrf/pull/7653.

File: samples/bluetooth/peripheral_fast_pair/README.rst [hunk 1]
```diff
@@ -186,22 +191,28 @@ Test not discoverable advertising by completing `Testing`_ and the following add
    a. Go to :guilabel:`Settings` > :guilabel:`Google` > :guilabel:`Devices & sharing` (or :guilabel:`Device connections`) > :guilabel:`Devices` > :guilabel:`Saved devices`.
    #. Verify that the paired device is appearing on the list.
 
-#. Press **Button 1** to switch to the Fast Pair not discoverable advertising.
-   **LED 3** turns off.
+#. Press **Button 1** to switch to the Fast Pair not discoverable advertising show UI indication mode.
+   **LED 3** blinks rapidly.
+   If you want to test the Fast Pair not discoverable advertising hide UI indication mode, press **Button 1** again.
+   **LED 3** blinks slowly then.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_fast_pair/README.rst:197]:
```suggestion
   **LED 3** starts blinking slowly.
```

File: samples/bluetooth/peripheral_fast_pair/README.rst [hunk 2]
```diff
@@ -186,22 +191,28 @@ Test not discoverable advertising by completing `Testing`_ and the following add
    a. Go to :guilabel:`Settings` > :guilabel:`Google` > :guilabel:`Devices & sharing` (or :guilabel:`Device connections`) > :guilabel:`Devices` > :guilabel:`Saved devices`.
    #. Verify that the paired device is appearing on the list.
 
-#. Press **Button 1** to switch to the Fast Pair not discoverable advertising.
-   **LED 3** turns off.
+#. Press **Button 1** to switch to the Fast Pair not discoverable advertising show UI indication mode.
+   **LED 3** blinks rapidly.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_fast_pair/README.rst:195]:
```suggestion
   **LED 3** starts blinking rapidly.
```

File: samples/bluetooth/peripheral_fast_pair/README.rst [hunk 3]
```diff
@@ -86,7 +86,11 @@ Buttons
 =======
 
 Button 1:
-   Toggles between Fast Pair discoverable and not discoverable advertising.
+   Toggles between three Fast Pair advertising modes:
+
+   * Fast Pair discoverable advertising.
+   * Fast Pair not discoverable advertising, show UI indication.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_fast_pair/README.rst:92]:
```suggestion
   * Fast Pair not discoverable advertising (with the show UI indication).
```

File: samples/bluetooth/peripheral_fast_pair/README.rst [hunk 4]
```diff
@@ -110,10 +114,11 @@ LED 2:
    * Off if there is no Central connected.
 
 LED 3:
-   Depending on the discoverability setting:
+   Depending on the Fast Pair advertising mode setting:
 
    * On if the device is Fast Pair discoverable.
-   * Off if the device is Fast Pair not discoverable.
+   * Blinks with 0.5 secs interval if the selected mode is Fast Pair not discoverable advertising, show UI indication.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_fast_pair/README.rst:120]:
```suggestion
   * Blinks with 0.5 secs interval if the selected mode is the Fast Pair not discoverable advertising with the show UI indication.
```

File: samples/bluetooth/peripheral_fast_pair/README.rst [hunk 5]
```diff
@@ -110,10 +114,11 @@ LED 2:
...
+   * Blinks with 1.5 secs interval if the selected mode is Fast Pair not discoverable advertising, hide UI indication.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_fast_pair/README.rst:121]:
```suggestion
   * Blinks with 1.5 secs interval if the selected mode is the Fast Pair not discoverable advertising with the hide UI indication.
```

File: samples/bluetooth/peripheral_fast_pair/README.rst [hunk 6]
```diff
@@ -186,22 +191,28 @@ Test not discoverable advertising by completing `Testing`_ and the following add
...
+
 #. Wait until the Fast Pair Provider is added to :guilabel:`Saved devices` on the second Android device:
 
    a. Go to :guilabel:`Settings` > :guilabel:`Google` > :guilabel:`Devices & sharing` (or :guilabel:`Device connections`) > :guilabel:`Devices` > :guilabel:`Saved devices`.
       The paired device appears on the list.
    #. If the device does not appear on the list, wait until the data is synced between phones.
 
 #. Move the second Android device close to the Fast Pair Provider.
-   A notification similar to the following one appears:
+   If the device is in the show UI indication mode, a notification similar to the following one appears:
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_fast_pair/README.rst:206]:
```suggestion
   If the device is in the show UI indication advertising mode, a notification similar to the following one appears:
```

File: samples/bluetooth/peripheral_fast_pair/README.rst [hunk 7]
```diff
@@ -186,22 +191,28 @@ Test not discoverable advertising by completing `Testing`_ and the following add
...
 
    .. figure:: /images/bt_fast_pair_not_discoverable_notification.png
       :scale: 50 %
       :alt: Fast Pair not discoverable advertising Android notification
 
-#. Tap on the notification to trigger the Fast Pair procedure.
+   If the device is in the hide UI indication mode, no notification appears.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_fast_pair/README.rst:212]:
```suggestion
   If the device is in the hide UI indication advertising mode, no notification appears.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 8]
```diff
@@ -134,6 +134,10 @@ Bluetooth samples
 
   * Added a workaround for nRF5340 revision 1 Errata 117.
 
+* :ref:`peripheral_fast_pair` sample:
+
+  * Added possibility of toggling between show/hide UI indication Fast Pair advertising modes.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:139]: Too many nouns :)
```suggestion
  * Added possibility of toggling the advertising modes.
    You can now toggle between discoverable and not discoverable modes (with show UI and hide UI indications).
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 9]
```diff
@@ -215,7 +219,9 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`bt_fast_pair_readme` library:
+
+  * Added option to set hide UI indication flag in Fast Pair not discoverable advertising data.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:224]:
```suggestion
  * Added option to set the flag for the hide UI indication in the Fast Pair not discoverable advertising data.
```

File: samples/bluetooth/peripheral_fast_pair/README.rst [hunk 10]
```diff
@@ -186,22 +191,28 @@ Test not discoverable advertising by completing `Testing`_ and the following add
...
+   The device advertises (you can check that in the nRF Connect for Mobile app), but do not want to be paired with.
+
+#. If the notification appears, tap on it to trigger the Fast Pair procedure.
```
Comment (greg-fer) [inline samples/bluetooth/peripheral_fast_pair/README.rst:215]:
```suggestion
#. In the show UI indication mode, when the notification appears, tap on it to trigger the Fast Pair procedure.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 11]
```diff
@@ -134,6 +134,11 @@ Bluetooth samples
 
   * Added a workaround for nRF5340 revision 1 Errata 117.
 
+* :ref:`peripheral_fast_pair` sample:
+
+  * Added possibility of toggling the advertising modes.
```
Comment (MarekPieta) [inline doc/nrf/releases/release-notes-changelog.rst:139]: Maybe: `Added possibility of toggling between show and hide UI indication in Fast Pair not discoverable advertising.`

File: doc/nrf/releases/release-notes-changelog.rst [hunk 12]
```diff
@@ -215,7 +220,9 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`bt_fast_pair_readme` library:
+
+  * Added option to set the flag for the hide UI indication in the Fast Pair not discoverable advertising data.
```
Comment (MarekPieta) [inline doc/nrf/releases/release-notes-changelog.rst:225]: `Extended API to allow setting the flag ...`

Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:225]: `option` may seem like reference to Kconfig option which could be a bit confusing

File: doc/nrf/releases/release-notes-changelog.rst (hunk 11 above)
Comment (MarekPieta) [reply doc/nrf/releases/release-notes-changelog.rst:139]: Toggling between discoverable and not discoverable advertising was introduced before.

**Review by greg-fer — APPROVED** [review]

**Review by MarekPieta — APPROVED** [review]

**Review by KAGA164 — APPROVED** [review]

**Pull request merged**